application/transaction/transaction_test.go

```diff
@@ -5,32 +5,30 @@ import (
 
 	"github.com/fabric8-services/fabric8-cluster/application/transaction"
 	"github.com/fabric8-services/fabric8-cluster/gormtestsupport"
-	"github.com/fabric8-services/fabric8-cluster/resource"
 
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 )
 
-type TestTransaction struct {
+type TestTransactionSuite struct {
 	gormtestsupport.DBTestSuite
 }
 
-func TestRunTransaction(t *testing.T) {
-	resource.Require(t, resource.Database)
-	suite.Run(t, &TestTransaction{DBTestSuite: gormtestsupport.NewDBTestSuite()})
+func TestRunTransactionSuite(t *testing.T) {
```

Comment:
I would simply name if `TestTransaction`, but again, it's just a matter of naming things

Reply:
Done in https://github.com/fabric8-services/fabric8-cluster/pull/24/commits/9127aea20f12b3acd503ae8b47efbec3c1852d8d